templates/silicon/index.html

```diff
@@ -0,0 +1,550 @@
+{% extends "silicon/base_silicon.html" %}
+
+{% block title %}Silicon Programme{% endblock %}
+
+{% block meta_description %}
+  Unlock new opportunities with optimised secure open source solutions
+{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1nyiz4I1GqBiXe_SKpDzho3yv20LKyYpyhHh0fvvUrOI/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-shape-dark is-deep">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-8">
+      <h1 class="p-heading--2 u-no-margin--bottom" style="font-weight: 500;">Silicon programme</h1>
+      <p class="p-heading--2" style="font-weight: 100;">
+        Unlock new opportunities with optimised secure open source solutions
+      </p>
+      <p>Canonical is a trusted partner for silicon vendors with open source solutions. Canonical infrastructure software solutions with custom silicon optimizations, security, maintenance and support have helped silicon companies reduce cost, improve software performance and create additional business opportunities across cloud and edge markets.</p>
+      <p>
+        <a class="p-button--positive" href="https://canonical.com/partners/become-a-partner">
+          Become a partner
+        </a>
+      </p>
+    </div>
+    <div class="col-4 u-hide--medium u-hide--small u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/f20789f7-silicon-5C.svg",
+        alt="silicon chip illustration",
+        width="336",
+        height="168",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <div class="p-logo-section">
+      <p class="p-logo-section__title">
+        A selection of our silicon partners
+      </p>
+      <div class="p-logo-section__items u-sv-3">
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/1ac4edd4-NVIDIA.png",
+            alt="Nvidia logo",
+            width="292",
+            height="290",
+            hi_def=True,
+            loading="auto",
```

Comment:
```suggestion
            loading="lazy",
```
Usually we we only use auto for the header image and the rest can be lazy. However, these logos are visible on page load, so I am not sure. @lyubomir-popov What is the proper practice here?

Reply:
So I checked other places where images appear right under the headr/takover and they were lazy loaded so I'm going to make them lazy loaded.